include/lsst/meas/base/ApertureFlux.h

```diff
@@ -210,9 +210,9 @@ class ApertureFluxAlgorithm : public SimpleAlgorithm {
 };
 
 /**
- *  A Result struct for running an aperture flux algorithm with a single radius.
+ *  A Result struct for running an aperture instFlux algorithm with a single radius.
  *
- *  This simply extends FluxResult to add the appropriate error flags for aperture fluxes.
+ *  This simply extends FluxResult to add the appropriate error flags for aperture instFluxes.
```

Comment:
Is the flag for instFlux different than the flag for flux? If they share a flag then this change seems a bit off